2020-07-21 14:09:05,850 ERROR: Exception on /reset_password_request [POST] [in c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\routes.py", line 180, in reset_password_request
    send_password_reset_email(user)
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\email.py", line 18, in send_password_reset_email
    html_body=render_template('email/reset_password.html',user=user,token=token))
  File "C:\Users\lenovo\Desktop\microblog\Python-Projects\Microblog\app\email.py", line 9, in send_email
    mail.send(msg)
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask_mail.py", line 491, in send
    with self.connect() as connection:
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask_mail.py", line 144, in __enter__
    self.host = self.configure_host()
  File "c:\users\lenovo\desktop\microblog\python-projects\microblog\venv\lib\site-packages\flask_mail.py", line 163, in configure_host
    host.starttls()
  File "C:\Users\lenovo\anaconda3\envs\cs231n\lib\smtplib.py", line 752, in starttls
    "STARTTLS extension not supported by server.")
smtplib.SMTPNotSupportedError: STARTTLS extension not supported by server.
